tests/unit/test_backends.py

```diff
@@ -242,3 +242,14 @@ def testParsePageToken(self):
         goodPageToken = "12:34:567:8:9000"
         parsedToken = backend._parsePageToken(goodPageToken, 5)
         self.assertEqual(parsedToken[2], 567)
+
+    def testParseIntegerArgument(self):
+        args = {"good": "1", "negative": "-1", "bad": "A", "badfloat": "0.98"}
```

Comment:
Why not split the success and failure cases into two lists rather than using the keys here? Using hard-coded strings for control flow is always tricky...